base/file.jl

```diff
@@ -540,10 +541,16 @@ end # os-test
 
 
 """
-    tempname()
+    tempname(parent=tempdir(); cleanup=true) -> String
 
 Generate a temporary file path. This function only returns a path; no file is
-created. The path is likely to be unique, but this cannot be guaranteed.
+created. The path is likely to be unique, but this cannot be guaranteed. When
```

Comment:
Is it worth pointing out here that while the name is not garenteed to be unique,
it is garenteed not to collice with an existing filename?